src/icons/fileIcons.ts

```diff
@@ -2407,5 +2407,9 @@ export const fileIcons: FileIcons = {
       name: 'snapcraft',
       fileNames: ['snapcraft.yaml', 'snapcraft.yml'],
     },
+    {
+      name: 'container',
+      fileNames: ['devcontainer.json'],
```

Comment:
Can we also add the foldername ".devcontainer"?

```suggestion
      fileNames: ['.devcontainer/devcontainer.json'],
```

Reply:
And the truth is, it will be better that way